install/kubernetes/helm/deployIstioHelmCharts.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+echo "Deploying Istio..."
+
+echo "Creating RBAC rules for Istio..."
+
+helm install istio-rbac --name istio-rbac
+retCode=$?
+if [ $retCode != '0' ]; then
+  echo "Creating of RBAC rules failed, exiting..."
+  exit 1 
+fi
+
+echo "Creating a namespace for Istio..."
+
+helm install istio-namespace --name istio-namespace
+retCode=$?
+if [ $retCode != '0' ]; then
+  echo "Creating of namespace failed, exiting..."
+  exit 2
+fi
+
+echo "Creating CRD objects for Istio..."
+
+helm install istio-crds --name istio-crds
+retCode=$?
+if [ $retCode != '0' ]; then
+  echo "Creating of CRDs objects failed, exiting..."
+  exit 3
+fi
+
+echo "Deploying Istio..."
+
+helm install istio --name istio
```

Comment:
I found myself consistently wanting to do the following:
helm install istio --name istio --set istio.ingress.use_nodeport=true

perhaps taking an argument to this last step in the deploy process would be helpful.

Reply:
I added helmExtraArgs parameter which can be used to pass to helm any additional arguments which must be recognizable by helm.
so in your case then you would use
deployIstioHelmCharts.sh "--set istio.ingress.use_nodeport=true"